src/simple-todos/step12/imports/ui/App.svelte

```diff
@@ -22,24 +21,25 @@
 
     if (user) {
 
-        isLoading = !handler.ready();
+      isLoading = !handler.ready();
 
-        const userFilter = { userId: user._id };
-        const pendingOnlyFilter = { ...hideCompletedFilter, ...userFilter };
+      const userFilter = { userId: user._id };
+      const pendingOnlyFilter = { ...hideCompletedFilter, ...userFilter };
 
 
-        tasks = TasksCollection.find(
-                hideCompleted ? pendingOnlyFilter : userFilter,
-                { sort: { createdAt: -1 } }
-            ).fetch();
+      getTasks = user
+        ? TasksCollection.find(
+          hideCompleted ? pendingOnlyFilter : userFilter,
+          { sort: { createdAt: -1 } }
+        ).fetchAsync()
+        : [];
 
-        incompleteCount = TasksCollection.find(pendingOnlyFilter).count();
-
-        pendingTasksTitle = `${
-          incompleteCount ? ` (${incompleteCount})` : ''
-        }`;
+      getCount = user
+        ? TasksCollection.find(pendingOnlyFilter).countAsync()
+        : 0;
```

Comment:
These two ternaries are unnecessary since it checked if there is a user on line 22